corehq/apps/app_manager/management/commands/migrate_case_search_prompt_itemset_ids.py

```diff
@@ -18,14 +19,23 @@ def migrate_app(self, app_doc):
         should_save = False
         for module in app_doc.get('modules', []):
             if module.get('search_config'):
-                for prop in module.get('search_config').get('properties', []):
+                properties = module.get('search_config').get('properties')
+                if not isinstance(properties, list):
```

Comment:
One doc on staging has `properties` set to a single `CaseSearchProperty` instead of a list. This makes the app unusable (it can't be wrapped) so it's fine to skip.